### PR TITLE
Wallet, Account creation, Web 3 Browser and Messenger pages

### PR DESCRIPTION
**Pull request by NistorCristian**
I've created 4 pages that were requested by @j-zerah . Wallet, Account creation, Web 3 Browser and Messenger. The pull request also has some changes for homepage that were in Figma.

Signed-off-by: Nistor Cristian <contact@nistorcristian.com>

### CODE REVIEW

File: themes/navy/layout/index.ejs [hunk 1]
```diff
@@ -1,194 +1,254 @@
...
+	</section>
+	<!--END-HOME-ABOUT-->
+	<!--HOME-NETWORK-->
+	<section>
+		<div class="max-w-screen-2xl mx-auto lg:py-48 lg:px-12">
+			<div class="grid md:grid-cols-1">
+				<div class="flex flex-col justify-center text-white lg:xl-48 px-8 items-center py-32 bg-primary-base">
+					<h3 class="font-display text-4xl xl:text-5xl 2xl:text-6xl text-center"><%- __('site.home.status-network.title') %></h3>
+					<p class="text-primary-200 font-thin text-xl max-w-screen-md px-12 text-center pt-12"><%- __('site.home.status-network.description') %></p>
+					<a href="https://dap.ps/" target="_blank" class="group inline-flex group items-center text-white font-semibold font-special mt-12">
```
Comment (jakubgs) [inline themes/navy/layout/index.ejs:245]: This should not be `dap.ps`.

File: themes/navy/source/js/main.js [hunk 2]
```diff
@@ -816,6 +816,240 @@ $(document).ready(function($) {
...
+    duration: 0,
+    viewFactor: 1,
+    afterReveal: function(){
+
+      privateAndSecureAnimation();
+
+    }
+  });
+
+  function privateAndSecureAnimation(){
```
Comment (jakubgs) [inline themes/navy/source/js/main.js:830]: Could you please add comments for each one of these methods and what they do?

File: themes/navy/source/js/main.js [hunk 3]
```diff
@@ -816,6 +816,240 @@ $(document).ready(function($) {
     }
   });
 
+  ScrollReveal().reveal('.private-and-secure', {
```
Comment (jakubgs) [inline themes/navy/source/js/main.js:819]: What does this do? Commen?

File: themes/navy/source/js/main.js [hunk 4]
```diff
@@ -816,6 +816,240 @@ $(document).ready(function($) {
...
+    duration: 0,
+    viewFactor: .5,
+    afterReveal: function(){
+
+      homeAllAnimation();
+
+    }
+  });
+
+  function homeAllAnimation(){
```
Comment (jakubgs) [inline themes/navy/source/js/main.js:926]: Moar comments needed.

File: themes/navy/layout/messenger.ejs [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+					<h1 class="leading-tight font-display text-6xl font-medium xl:text-7xl 2xl:text-8xl"><%- __('site.messenger.intro.title') %></h1>
+					<p class="text-gray-600 mt-12 mb-16 text-xl xl:text-4xl font-display font-normal leading-normal">
+						<%- __('site.messenger.intro.description') %>
+					</p>
+					<div class="flex items-center -ml-12 mt-12">
+						<a href="<%- get_build_url('official', 'APPSTORE') %>" id="official:ios" class="ml-12 app-download"><%- image_tag("/img/app-store.svg") %></a>
+						<a href="<%- get_build_url('official', 'PLAYSTORE') %>" id="official:apk" class="ml-12 app-download"><%- image_tag("/img/google-play.svg") %></a>
+						<a href="<%- get_build_url('official', 'DIRECT_APK') %>" id="official:apk" class="ml-12 app-download"><%- image_tag("/img/apk.svg") %></a>
+					</div>
+				</div>
```
Comment (jakubgs) [inline themes/navy/layout/messenger.ejs:17]: Could we just make this whole `<div>` a partial/template to reuse in the index page and new pages?

**Comment by jakubgs** [comment]
You need to rebase the branch.

**Comment by NistorCristian** [comment]
7d3d95a fixes #568 , #567 , #567 , #566 , #565 , #564 , #563 , #562 , #561 , #560 , #559 , #558 , #540

**Comment by j-zerah** [comment]
@NistorCristian can you please review the changes required, so I can then review in dev

**Comment by jinhojang6** [comment]
@NistorCristian

Can we sort out the rebase and merge this branch this week? I need to have this PR confirmed and merged with the develop branch to start translations.

**Comment by jakubgs** [comment]
You're going father and father away from `develop`. Please rebase your branch on `develop`.

**Comment by jakubgs** [comment]
Rebasing your branch is pretty simple. All that's doing is updating the point from which you started your own feature branch by attempting to apply each commit that has been added on top of parent branch since your own branch was created until it reaches it's `HEAD`(most recent commit).

Like so:
git_rebase

You can simply do it by running this from your own branch:
```
git rebase develop
```
Which will attempt to apply each commit that you don't have from the parent branch, and if anything is wrong it will stop and give you options. Most involve editing the file to look like you want it to and then running `git rebase --continue` to go on to another commit.

Read up: https://git-scm.com/book/en/v2/Git-Branching-Rebasing

**Comment by jinhojang6** [comment]
@NistorCristian 

Please review this pull request: https://github.com/status-im/status.im/pull/579

**Comment by jakubgs** [comment]
Closing this as #579 has the rebased version of these changes, but I'd like my comments to be addressed anyway.

**Pull request closed without merging**